docs/user-guide/building-assistants.rst

```diff
@@ -144,6 +146,23 @@ You can now train a first model and test the bot, by running the following comma
 
 This bot should now be able to reply to the intents we defined consistently, and in any order.
 
+For example, it should be able to do:
+
+.. code-block:: bash
+
+    Bot loaded. Type a message and press enter (use '/stop' to exit): 
+    Your input ->  bye!
+    Bye!
+    Your input ->  thanks!
+    No worries!
+    Your input ->  hi!
+    Hi
+    Your input ->  goodbye
+    Bye!
+    Your input ->  thank you
+    No worries!
```

Comment:
any reason we didn't go for a screenshot here btw?

Reply:
I wasn't sure if a screenshot of the commandline was appropriate, but can put it in if it would be better

Reply:
what do you think? does it look funny?

Reply:
nah let's just leave it, it's fine!